doc/source/tutorial/stats/hypothesis_fisher_exact.md

```diff
@@ -0,0 +1,98 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+```{eval-rst}
+.. jupyterlite:: ../../_contents/hypothesis_fisher_exact.ipynb
+   :new_tab: True
+```
+
+(hypothesis_fisher_exact)=
+# Fisher's exact test
+
+In [^1], the effective dose of acetazolamide for the prophylaxis of acute
```

Comment:
Something to note about the example notebooks is that reference numbers now appear as footnotes. Ideally, they would be normal sized text.

Reply:
Yeah, this is effectively a footnote. As far as I'm aware, we can fix this in two ways:
1. add a css element that fixes the rendering in scipy.css, i.e. fixing
    ``` 
    a.footnote-reference {
        font-size: small;
        vertical-align: super;
    }
    ```
    This would however change the footnote rendering for all pages, so not sure if that would have unwanted effects;
2. Use simple links insted of footnotes, and manually add a references section to the end of each page.

What do you think?

Reply:
Let's leave it for now.